cardano-db-tool/src/Cardano/Db/Tool/Validate/Balance.hs

```diff
@@ -15,10 +15,10 @@ import           Cardano.Chain.Common (CompactAddress, Lovelace, decodeAddressBa
                    toCompactAddress, unsafeGetLovelace)
 import qualified Cardano.Chain.UTxO as Byron
 
+import qualified Cardano.Ledger.Core as Core
```

Comment:
The qualified imports in this project are far from optimal and I have been meaning to fix them.

As a start in the right direction, can we import that as `Ledger` ?